newamericadotorg/assets/react/report/components/BasicHeader.js

```diff
@@ -0,0 +1,36 @@
+import './BasicHeader.scss';
+import React, { Component } from 'react';
+
+class BasicHeader extends Component {
+  render() {
+    let { report } = this.props;
+
+    return (
+      <React.Fragment>
```

Comment:
Nitpick: I don't think we need to return a fragment here. Per my understanding, a fragment is needed when you cannot for whatever reason wrap the output in a single element. You have a single outer div here, so I think it is safe to remove  <React.Fragment>. Please correct me if I am wrong (I learned on a newer version of React, so maybe that was not the case previously).

Reply:
Ah yup, you're right. Left over cruft from a copy/pasta.